doc/DOWNLOADS.md

```diff
@@ -60,20 +60,6 @@ To install Nicotine+ on FreeBSD, run the following:
 pkg install py-nicotine-plus
 ```
 
```

Comment:
Suggest insert line with link to #pip section:

"`If Nicotine+ is not packaged for your system, the current stable version can be installed using pip ([see below])(DOWNLOADS.md#pip).`"

Or some other words as you deem appropriate.